src/main/java/fi/dy/masa/litematica/scheduler/tasks/TaskPasteSchematicPerChunkCommand.java

```diff
@@ -549,7 +549,7 @@ protected void placeBlockViaClone(BlockPos pos, BlockState state, BlockEntity be
     protected BlockPos placeNbtPickedBlock(BlockPos pos, BlockState state, BlockEntity be,
                                            World schematicWorld, ClientWorld clientWorld)
     {
-        double reach = this.mc.interactionManager.getReachDistance();
+        double reach = 5.0; //this.mc.interactionManager.getReachDistance();
```

Comment:
I was unable to find the new location for this in the class so hard-coded 5

Reply:
The best replacement would probably be PlayerEntity::getBlockInteractionRange

Reply:
thank you!